atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/DbKvs.java

```diff
@@ -105,6 +107,7 @@
     private final DdlConfig config;
     private final DbTableFactory dbTables;
     private final SqlConnectionSupplier connections;
+    private final OracleTableNameGetter oracleTableNameGetter;
```

Comment:
This shouldnt be a field for ``Dbkvs``, how about only passing the oracleTableNameGetter to ``DbkvsGetRanges``?